packages/land/contracts/common/ERC721BaseToken.sol

```diff
@@ -317,7 +317,7 @@ abstract contract ERC721BaseToken is IContext, IERC721, IERC721Errors, WithSuper
         if (owner == address(0)) {
             revert ERC721NonexistentToken(tokenId);
         }
-        if (from != owner) {
+        if (from != owner && !operatorEnabled) {
```

Comment:
This is ok? operatorEnabled must be always used with another a call to _getOperator: `(operatorEnabled && _getOperator(tokenId) == msgSender)`